guides/manual-testing.md

```diff
@@ -1,37 +1,59 @@
-### Manually testing the Scaffolding tool
+# Manually Testing the Scaffolding Tool
 
-#### Ensuring you're using the right Scaffolding version
+## Testing with the Current Local Version
 
-With the Scaffolding repository checked out, switch to the Git revision you want to test.
+To test the current local version of the Scaffolding tool:
 
-Start the Nix development environment for the Scaffolding tool using:
+1. Check out the Scaffolding repository and switch to the Git revision you want to test.
 
-```shell
-nix develop --override-input "versions/scaffolding" "path://$PWD"
-```
+2. Start the Nix development environment:
 
-This will give you a shell which includes the Scaffolding built from your local repository.
+   ```shell
+   nix develop
+   ```
+   This shell includes the Scaffolding tool built from your current local repository.
 
-Stay in the current Nix shell and change directory to somewhere that you want to put your test project. 
-For example, `cd /tmp`. You're now ready to scaffold a new app with `hc-scaffold web-app`.
+3. Change to a directory where you want to create your test project, e.g.:
 
-When following the instructions that the Scaffolding outputs, ignore `nix develop` and instead run:
+   ```shell
+   cd /tmp
+   ```
 
-```shell
-nix develop --override-input "versions/scaffolding" <path-to-local-scaffolding-clone>
-```
+4. Scaffold a new app:
+
+   ```shell
+   hc-scaffold web-app
+   ```
+
+5. Follow the Scaffolding tool's output instructions, but skip the `nix develop` step to ensure you continue using the locally built `hc-scaffold`.
 
-Which will ensure that you keep using the updated Scaffolding tool inside the scaffolded hApp environment.
+6. Run tests or launch your hApp using `nix develop` as needed.
```

Comment:
Oh I see, so you'd do your scaffolding in the shell that has the custom hc-scaffold but use another shell to do your app commands - could that be made more explicit in the steps?